### ISSUE TITLE
Pod spec does not have 2.0.0 available

### ISSUE THREAD

**Issue by matadan** [tid-kijyun/Kanna#100]
### Description:

The readme says to use

`pod 'Kanna', '~> 2.0.0'`

in the podfile but there is no spec found for this.
### Installation method:
- [ ] Carthage
- [x] CocoaPods
- [ ] Manually
### Library version:
- [ ] v1.1.1
- [ ] v1.0.6
- [ ] v1.0.2
- [x] v2.0.0: (Please fill in the version you are using.)
### Xcode version:
- [ ] 8.0 beta (Swift 3.0)
- [ ] 8.0 beta (Swift 2.3)
- [ ] 7.3.1
- [x] 8.0: (Please fill in the version you are using.)

**Comment by balazsnemeth** [comment]
Same for me. 

```
Unable to satisfy the following requirements:
- `Kanna (~> 2.0.0)` required by `Podfile`
```

I performed `pod repo update`.

**Comment by jitae1126** [comment]
> pod 'Kanna', :git => 'https://github.com/tid-kijyun/Kanna', :branch => 'master'

This lets you install 2.0.0, but I still get compile errors (in a new project)

**Comment by nunogoncalves** [comment]
Yeap! Me too. :(

**Comment by nunogoncalves** [comment]
There's the swift3.0 branch #96. We can use it for now.

`pod 'Kanna', git: 'https://github.com/tid-kijyun/Kanna.git', branch: 'swift3.0'`

**Comment by jitae1126** [comment]
Thanks @nunogoncalves ! The master had a more recent commit so I assumed I should have used it but the swift3.0 branch works perfectly.

**Comment by picopeak** [comment]
I still have issue around the code below. It says the XML returned doesn't match any available overloads. Any idea?

public func XML(_ xml: String, encoding: String.Encoding, option: ParseOption = kDefaultXmlParseOption) -> XMLDocument? {
    return XML(xml: xml, url: nil, encoding: encoding, option: option)
}

// NSData
public func XML(_ xml: Data, url: String?, encoding: String.Encoding, option: ParseOption = kDefaultXmlParseOption) -> XMLDocument? {
    if let xmlStr = NSString(data: xml, encoding: encoding.rawValue) as? String {
        return XML(xml: xmlStr, url: url, encoding: encoding, option: option)
    }
    return nil
}

public func XML(_ xml: Data, encoding: String.Encoding, option: ParseOption = kDefaultXmlParseOption) -> XMLDocument? {
    return XML(xml: xml, url: nil, encoding: encoding, option: option)
}

// NSURL
public func XML(_ url: URL, encoding: String.Encoding, option: ParseOption = kDefaultXmlParseOption) -> XMLDocument? {
    if let data = try? Data(contentsOf: url) {
        return XML(xml: data, url: url.absoluteString, encoding: encoding, option: option)
    }
    return nil
}

**Comment by swimstaralex** [comment]
swift3.0 branch has compile errors for me, too. How to fix this?

**Comment by tid-kijyun** [comment]
I've just fixed this issue.

Please use Cocoapods 1.1.0 rc.

**Comment by tid-kijyun** [comment]
I just released v2.1.0.

```
pod 'Kanna', '~> 2.1.0'
```

If there is still a problem, please reopen this issue.
